internal/engine/image_build_and_deployer.go

```diff
@@ -115,7 +115,7 @@ func (ibd *ImageBuildAndDeployer) BuildAndDeploy(ctx context.Context, st store.R
 		}
 
 		q.SetResult(iTarget.ID(), store.BuildResult{Image: ref})
-		anyFastBuild = anyFastBuild || iTarget.IsFastBuild()
+		anyFastBuild = anyFastBuild || iTarget.MaybeFastBuildInfo() != nil
```

Comment:
It seems bad that `IsFastBuild` isn't accurate here - we should update or remove it.

Reply:
I think @dbentley was saying that it's desirable for `IsFastBuild()` to mean that it is a fast build and _only_ a fast build and not to overload it to mean "HasFastBuild" or "ContainsFastBuild".

Reply:
Yeah, I think the way we think of "Fast Build" is weird. We use this now in just one place (analytics_reporter.go); maybe make it use MaybeFastBuild and delete IsFastBuild. (We probably don't need to assert, in tiltfile_docker_compose_test.go, that a target is both staticbuild and not fastbuild)